WordPress/Jetpack/JetpackDebug.entitlements

```diff
@@ -12,6 +12,10 @@
 	<array>
 		<string>webcredentials:wordpress.com</string>
 		<string>applinks:apps.wordpress.com</string>
+		<string>applinks:wordpress.com</string>
+		<string>applinks:public-api.wordpress.com</string>
+		<string>applinks:links.wp.a8cmail.com</string>
+		<string>applinks:*.wordpress.com</string>
```

Comment:
Might be out of scope, but I wonder if we need `apps.wordpress.com` when we already have `*.wordpress.com` 🤔 

Not a big deal, the WordPress entitlements do the same, so it might be worth looking at both in a dedicated PR later on 👌 

https://github.com/wordpress-mobile/WordPress-iOS/blob/c4a83561d8794b22838f4baaa881a11316d6fb98/WordPress/WordPress.entitlements#L15-L18